dsn.go

```diff
@@ -186,6 +186,9 @@ func DSN(cfg *Config) (dsn string, err error) {
 			params.Add(k, *v)
 		}
 	}
+	if cfg.KeepSessionAlive {
+		params.Add("client_session_keep_alive", strconv.FormatBool(cfg.KeepSessionAlive))
```

Comment:
Can we stick to the variable name and naming conventions? I'd prefer calling this parameter `keepSessionAlive`.